benchmarking/bench.js

```diff
@@ -1,5 +1,7 @@
 import axios from "axios";
 
+const tokens = ["BTC", "ETH", "DAI", "AST"];
```

Comment:
Tokens will always be addresses so suggest we use those for testing.

Reply:
Changed